test/steps/crypto/1_given.js

```diff
@@ -135,20 +135,40 @@ export function aPassword() {
 	this.test.ctx.password = password;
 }
 
-export function anEncryptedPassphraseWithAnIV() {
-	const [encryptedPassphrase, iv] = getQuotedStrings(this.test.parent.title);
-	const cipherAndIv = {
-		cipher: encryptedPassphrase,
-		iv,
+export function anEncryptedPassphrase() {
+	const encryptedPassphrase = getFirstQuotedString(this.test.parent.title);
+	const encryptedPassphraseObject = {
+		iterations: 1,
+		salt: 'e8c7dae4c893e458e0ebb8bff9a36d84',
+		cipherText:
+			'c0fab123d83c386ffacef9a171b6e0e0e9d913e58b7972df8e5ef358afbc65f99c9a2b6fe7716f708166ed72f59f007d2f96a91f48f0428dd51d7c9962e0c6a5fc27ca0722038f1f2cf16333',
+		iv: '1a2206e426c714091b7e48f6',
+		tag: '3a9d9f9f9a92c9a58296b8df64820c15',
+		version: '1',
 	};
+	if (
+		typeof lisk.cryptography.parseEncryptedPassphrase.returns === 'function'
```

Comment:
Do we need to check if the function is already `stub` or not?

Reply:
Yes, at least as things are currently set up. In the `cryptography` suite we stub the `lisk-elements` functions, but in the actions suites we stub the `cryptography` module, and don't stub the `lisk-elements` functions.